genesyscloud/resource_genesyscloud_knowledge_knowledgebase.go

```diff
@@ -31,12 +58,26 @@ func getAllKnowledgeKnowledgebases(_ context.Context, clientConfig *platformclie
 			break
 		}
 
-		for _, knowledgeBase := range *knowledgeBases.Entities {
-			resources[*knowledgeBase.Id] = &ResourceMeta{Name: *knowledgeBase.Name}
+		entities = append(entities, *knowledgeBases.Entities...)
+
+		if knowledgeBases.NextUri == nil || *knowledgeBases.NextUri == "" {
```

Comment:
@carnellj-genesys Turns out this check is needed because the api response json doesn't include any field `NextUri` once it reaches the last page. If anything the check to see if the string is empty is probably superfluous , but I think I'll just leave it in to be safe